contrib/guix/libexec/build.sh

```diff
@@ -264,7 +259,7 @@ export PATH="${BASEPREFIX}/${HOST}/native/bin:${PATH}"
                 cp "${DISTSRC}/doc/README_windows.txt" "${DISTNAME}/readme.txt"
                 ;;
             *linux*)
-                cp "${DISTSRC}/doc/README.md" "${DISTNAME}/"
+                cp "${DISTSRC}/README.md" "${DISTNAME}/"
```

Comment:
why is this changed?

Reply:
Oh, I see c4a3c25ba11b20871699e570fc7b98a7b8472503

Reply:
Neither README make sense. The top level one talks about testing and quality assurance, which is meta for the tarball. The doc README has some dead links, but at least gives some more context. :shrug: